inference-engine/include/details/ie_so_pointer.hpp

```diff
@@ -208,17 +152,4 @@ class SOPointer {
 };
 
 }  // namespace details
-
-/**
- * @brief Creates a special shared_pointer wrapper for the given type from a specific shared module
- * @tparam T An type of object SOPointer can hold
- * @param name Name of the shared library file
- * @return A created object
- */
-template <class T>
-inline std::shared_ptr<T> make_so_pointer(const std::string & name) = delete;
-
-template <class T>
-inline std::shared_ptr<T> make_so_pointer(const std::wstring & name) = delete;
```

Comment:
this function is used in samples. Removing this right now breaks BW compatibility. 
Can we keep it for some time, but deprecate maybe?

Reply:
This function is used just for extension so leaved just extension version and added deprecation warning